### ISSUE TITLE
Array Relu, which is an input to the Conv operator producing the output array conv2d_1/BiasAdd, is lacking min/max data, which is necessary for quantization

### ISSUE THREAD

**Issue by shubham894** [tensorflow/tensorflow#36926]
Hi,
I am facing the following issue while converting my Frozen graph(.pb) to tflite 

Array Relu, which is an input to the Conv operator producing the output array conv2d_1/BiasAdd, is lacking min/max data, which is necessary for quantization. If accuracy matters, either target a non-quantized output format, or run quantized training with your model from a floating point checkpoint to change the input graph to contain min/max information. If you don't care about accuracy, you can pass --default_ranges_min= and --default_ranges_max= for easy experimentation.
Fatal Python error: Aborted

Current thread 0x00007f637125f740 (most recent call first):
  File "/home/shubhamsingh/miniconda3/lib/python3.7/site-packages/tensorflow/lite/toco/python/toco_from_protos.py", line 33 in execute
  File "/home/shubhamsingh/miniconda3/lib/python3.7/site-packages/absl/app.py", line 250 in _run_main
  File "/home/shubhamsingh/miniconda3/lib/python3.7/site-packages/absl/app.py", line 299 in run
  File "/home/shubhamsingh/miniconda3/lib/python3.7/site-packages/tensorflow/python/platform/app.py", line 40 in run
  File "/home/shubhamsingh/miniconda3/lib/python3.7/site-packages/tensorflow/lite/toco/python/toco_from_protos.py", line 59 in main
  File "/home/shubhamsingh/miniconda3/bin/toco_from_protos", line 8 in <module>
Aborted (core dumped)

I have used following script to convert the model
tflite_convert --output_file=mode_new_yoyo1.tflite --graph_def_file=frozen_model.pb --inference_type=QUANTIZED_UINT8 --input_arrays=input_1 --output_arrays=Dense1/Softmax,Dense2/Softmax --mean_values=0 --std_dev_values=255

I have used WideResnet model with two output node.

**Comment by gadagashwini** [comment]
@shubham894, Can you provide the Tensorflow version that you are using. Thanks!

**Comment by shubham894** [comment]
@gadagashwini  I am using Tensorflow 1.14.0 .

**Comment by liyunlu0618** [comment]
Are you converting a quantization-aware trained tf model? Can you try post-training integer [quantization](https://blog.tensorflow.org/2019/06/tensorflow-integer-quantization.html) instead?

**Comment by tensorflowbutler** [comment]
Hi There,

 We are checking to see if you still need help on this, as you are using an older version of tensorflow which is officially considered end of life . We recommend that you upgrade to the latest 2.x version and let us know if the issue still persists in newer versions. Please open a new issue for any help you need against 2.x, and we will get you the right help. 

 This issue will be closed automatically 7 days from now. If you still need help with this issue, please provide us with more information.
